### ISSUE TITLE
Configure two different switches in mininet simple_routing

### ISSUE THREAD

**Issue by huniyaarif1** [p4lang/p4factory#191]
I want a topology that is connecting hosts with two different switches l1 and l2 which are also connected to each other. They have different configurations, match+action tables and flow entries. I can add entries in one switch l1 using run_add_demo_entries. I have given a different path to the switch l2, but how do i add entries to this switch only?
lb1 = self.addSwitch('l2',
                                sw_path = nlb_path, //my custom path
                                thrift_port = thrift_port,
                                pcap_dump = True)

**Comment by antoninbas** [comment]
This repository is deprecated, please take a look at this thread: https://github.com/p4lang/p4factory/issues/190

**Comment by jafingerhut** [comment]
@antoninbas - If that is true, you will probably save everyone time in the long run if someone edits the README.md for this repo to say that it is deprecated, very near the beginning.

**Comment by antoninbas** [comment]
@jafingerhut I agree and I did see your email to the p4-dev mailing list. I do want to provide p4lang users with a detailed "How to get started" guide, but I haven't had time to get to it yet.
